src/lib/y2storage/btrfs_raid_level.rb

```diff
@@ -27,15 +28,37 @@ module Y2Storage
   #
   # This is a wrapper for the Storage::BtrfsRaidLevel enum
   class BtrfsRaidLevel
+    include Yast::I18n
+    extend Yast::I18n
+
     include StorageEnumWrapper
 
     wrap_enum "BtrfsRaidLevel"
 
-    # Returns human readable representation of enum which is already translated
+    TRANSLATIONS = {
+      default: N_("Default"),
+      single:  N_("SINGLE"),
```

Comment:
Likely `SINGLE` and `DUP` deserve a comment for the translators. Something like "This is one of the RAID modes used by Btrfs, likely to be left as-is".